Footer final

RuslanODUA opened the pull request.

RuslanODUA committed:
edit footer

edit class suptitle --> pre-title
edit _footer.scss

RuslanODUA committed:
Update _footer.scss

RuslanODUA committed:
Update _footer.scss

shmetanelena reviewed (commented):
1 на соцсети марджин для первой иконки не нужен, можно исправить так 
.networks__item:not(:first-child) {
  margin-left: 30px;
}

2 Цвет фона футера розовый

3 Нужно поставить марджин топ на 
<div class="footer__title">
На общие классы не ставим свои отступы

RuslanODUA committed:
add class

add class footer.html
title__indent
pre-title__indent
update _footer.scss

RuslanODUA committed:
linear-gradient

shmetanelena commented on src/partials/footer.html:
```diff
@@ -1,4 +1,5 @@
 <footer class="footer container">
+  <div class=""></div>
```
Что это за div?
Есть следующая проблема, заливка футера не по всей ширине контейнера, за это отвечает класс container,  но у тебя не срабатывает, его нужно использовать следующим  образом: 
footer class="footer">
<div class="container"> 
......
</div>
</footer>

shmetanelena commented on src/sass/layout/_footer.scss:
```diff
@@ -12,16 +12,18 @@
+  .title__indent {
+    margin-top: 17px;
+  }
   .title {
```
Обратила внимание, что ты переопределяешь title
Он у нас общий на весь проект, его лучше не трогать.
Необходимые свойства делать через расширение, как ты это уже  и так делаешь через title__indent.

shmetanelena commented on src/sass/layout/_footer.scss:
```diff
@@ -62,14 +64,17 @@
   }
-  .pre-title {
+  .pre-title__indent {
     margin-top: 117px;
```
Верхний отступ нужно сделать для всего footer__title.
У тебя этот margin пробил родителя

shmetanelena reviewed (changes requested):
Также добавь полосочку над копирайтом по всей ширине контейнера

RuslanODUA committed:
Update footer.html

RuslanODUA committed:
Update _footer.scss

RuslanODUA committed:
Merge branch 'main' into footer-final

RuslanODUA committed:
add class

add class container

RuslanODUA committed:
Update _footer.scss

RuslanODUA committed:
Update _footer.scss

RuslanODUA committed:
Update _footer.scss

RuslanODUA committed:
Update footer.html

add social network

RuslanODUA committed:
rename  update

rename symbol-defs.svg --> networks-icon.svg

update <a> add rel="noopener noreferrer"

RuslanODUA committed:
Update _footer.scss

update class copany
add margin-bottom: 10px;

RuslanODUA committed:
Update _footer.scss

RuslanODUA committed:
Update _footer.scss

RuslanODUA committed:
Update footer.html

del class lin

RuslanODUA committed:
new class

add class copyright file footer.html
edit file _footer.scss

shmetanelena committed:
use common titles

shmetanelena reviewed (approved):
Добавила свои изменения по использованию общего title

shmetanelena committed:
Merge branch 'main' into footer-final

shmetanelena committed:
renamed networks-icon to symbol-defs

shmetanelena committed:
add last end of line
